wiki/Article_Styling_Criteria/en.md

```diff
@@ -556,7 +586,7 @@ Where `<input>` is the file name to be compressed and `<output>` is the compress
 
 *Notice: All image file extensions must use lowercase letters, otherwise they will fail to load!*
 
-Image file names must never use spaces (use underscores (`_`) or hyphens (`-`) instead). When naming an image, the file name should be meaningful or descriptive but short. Never use timestamps or long hashes for the file name.
+Image file names must not use spaces (use underscores (`_`) or hyphens (`-`) instead). When naming an image, the file name should be meaningful or descriptive but short. Never use timestamps or long hashes for the file name.
```

Comment:
Can you prefer a character for the spacing to resolve #1996.

Reply:
Alright, we'll go with hyphens, for [this reason](https://github.com/ppy/osu-wiki/issues/1996#issuecomment-457861162)?

Reply:
Also the OP's reason, yeah.